content/docs/ops/setup/injection/index.md

```diff
@@ -11,14 +11,19 @@ time. Injection can be scoped to particular sets of namespaces using
 the webhooks `namespaceSelector` mechanism. Injection can also be
 enabled and disabled per-pod with an annotation.
 
-Whether or not a sidecar is injected is depends on three things:
+Whether or not a sidecar is injected depends on three pieces of configuration and two limitations:
 
+Configuration:
 * webhooks `namespaceSelector`
 * default `policy`
 * per-pod override annotation
 
+Limitations:
+* sidecars cannot be injected in the `kube-system` or `kube-public` namespaces
```

Comment:
what about istio-system namespace?

Reply:
My reading of the code didn't show any limitations on `istio-system`, but happy to add it